### PR TITLE
remove ordering clause that causes oracle compatibility issue

### PR DESCRIPTION
**Pull request by meara**
# Release Notes

Remove order clause from query that causes an oracle compatibility issue.

### CODE REVIEW

File: app/models/order_detail.rb [hunk 1]
```diff
@@ -171,7 +171,6 @@ def self.in_dispute
     where("dispute_at IS NOT NULL")
       .where(dispute_resolved_at: nil)
       .where("order_details.state != ?", "canceled")
-      .order("dispute_at")
```
Comment (jhanggi) [inline app/models/order_detail.rb]: Does this ordering still get applied where it's needed (i.e. the controller)?

Comment (meara) [reply app/models/order_detail.rb]: Added that back in.

**Comment by jhanggi** [comment]
Unclear if the CI failure is legit or not. I would suspect, not.

**Comment by meara** [comment]
That test isn't failing locally so I think should be fine on rebuild.

**Pull request merged**